<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>882</width>
    <height>573</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>861</width>
      <height>411</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QFrame" name="frame">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QSlider" name="horizontalSlider">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>450</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>490</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Speed</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>430</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Start animation</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>480</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>430</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Vaisseau en marche</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>430</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Arrêt du vaisseau</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>470</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Switch Marche avant/arriere</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>882</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAffichage">
    <property name="title">
     <string>Affichage</string>
    </property>
    <widget class="QMenu" name="menuTextures">
     <property name="title">
      <string>textures</string>
     </property>
     <addaction name="actionTexture"/>
     <addaction name="actionFil_de_fer"/>
     <addaction name="actionNone_2"/>
    </widget>
    <addaction name="menuTextures"/>
    <addaction name="actionOrbite"/>
    <addaction name="actionEclairage"/>
    <addaction name="actionAxes"/>
    <addaction name="actionAxe_XYZ"/>
   </widget>
   <widget class="QMenu" name="menuFiltrage">
    <property name="title">
     <string>Filtrage</string>
    </property>
    <widget class="QMenu" name="menuMag_filter">
     <property name="title">
      <string>Mag filter</string>
     </property>
     <addaction name="actionLinear"/>
     <addaction name="actionNearest_2"/>
    </widget>
    <widget class="QMenu" name="menuMin_filter">
     <property name="title">
      <string>Min filter</string>
     </property>
     <addaction name="actionLinear_mipmap_linear"/>
     <addaction name="actionNearest"/>
    </widget>
    <addaction name="menuMag_filter"/>
    <addaction name="menuMin_filter"/>
   </widget>
   <widget class="QMenu" name="menuVaisseau">
    <property name="title">
     <string>Vaisseau</string>
    </property>
    <widget class="QMenu" name="menuTrajectoire">
     <property name="title">
      <string>trajectoire</string>
     </property>
     <addaction name="actionSpline"/>
     <addaction name="actionDroite"/>
    </widget>
    <addaction name="actionBouclier"/>
    <addaction name="menuTrajectoire"/>
   </widget>
   <addaction name="menuAffichage"/>
   <addaction name="menuFiltrage"/>
   <addaction name="menuVaisseau"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNone">
   <property name="text">
    <string>None</string>
   </property>
  </action>
  <action name="actionTexture">
   <property name="text">
    <string>Texture</string>
   </property>
  </action>
  <action name="actionFil_de_fer">
   <property name="text">
    <string>Fil de fer</string>
   </property>
  </action>
  <action name="actionActiv">
   <property name="text">
    <string>activé</string>
   </property>
  </action>
  <action name="actionOrbite">
   <property name="text">
    <string>Orbite</string>
   </property>
  </action>
  <action name="actionEclairage">
   <property name="text">
    <string>Eclairage</string>
   </property>
  </action>
  <action name="actionNearest">
   <property name="text">
    <string>Nearest</string>
   </property>
  </action>
  <action name="actionLinear_mipmap_linear">
   <property name="text">
    <string>Linear</string>
   </property>
  </action>
  <action name="actionLinear">
   <property name="text">
    <string>Linear</string>
   </property>
  </action>
  <action name="actionNearest_2">
   <property name="text">
    <string>Nearest</string>
   </property>
  </action>
  <action name="actionNone_2">
   <property name="text">
    <string>None</string>
   </property>
  </action>
  <action name="actionAxes">
   <property name="text">
    <string>Axes des planetes</string>
   </property>
  </action>
  <action name="actionBouclier">
   <property name="text">
    <string>bouclier</string>
   </property>
  </action>
  <action name="actionAxe_XYZ">
   <property name="text">
    <string>Axe XYZ</string>
   </property>
  </action>
  <action name="actionSpline">
   <property name="text">
    <string>Spline</string>
   </property>
  </action>
  <action name="actionDroite">
   <property name="text">
    <string>Droite</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
